xapian-applications/omega/index_file.h

```diff
@@ -29,6 +29,7 @@
 #include <map>
 #include <string>
 #include <xapian.h>
+#include "worker.h"
```

Comment:
You don't need to include this here - only a forward declaration (i.e. `class Worker;`) is needed to be able to work with pointers or references to a type (if you dereference then a full declaration is needed).

The benefit of avoiding including headers when they aren't really needed is that fewer things need to rebuilt when that header is modified.  That's especially beneficial for an include in another header -

Reply:
Okay, I am going to change it.